FoxHound.Client/src/Posts/AddPost.tsx

```diff
@@ -29,13 +31,13 @@ const AddPost: React.FC = () => {
       const response = await Axios.post<number>(
         `${process.env.API_URL}/Post/Create`,
         {
-          blogId: 1,
+          blogdId: blogId,
```

Comment:
I think Jon mentioned this, not sure if you got to it yet. This should be blogId: blogId.